hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientReliableTopicProxy.java

```diff
@@ -338,4 +346,11 @@ private boolean terminate(Throwable failure) {
         }
     }
 
+    private Throwable getCauseIfAvailable(Throwable t) {
```

Comment:
There is a method in ExceptionUtil called 'peel' which afaik does the same thing.